charts/csm-application-mobility/templates/_helpers.tpl

```diff
@@ -0,0 +1,122 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "app-mobility.name" -}}
```

Comment:
Should the references here be renamed as well?

Reply:
Renamed these to be consistent. Verified that the same yaml is generated during `helm install` after renaming these to csm-application-mobility.